Geometry/GEMGeometryBuilder/src/GEMGeometryBuilderFromDDD.cc

```diff
@@ -34,243 +33,112 @@ GEMGeometryBuilderFromDDD::~GEMGeometryBuilderFromDDD()
 
 GEMGeometry* GEMGeometryBuilderFromDDD::build(const DDCompactView* cview, const MuonDDDConstants& muonConstants)
 {
-  std::string attribute = "ReadOutName"; // could come from .orcarc
-  std::string value     = "MuonGEMHits";    // could come from .orcarc
-  DDValue val(attribute, value, 0.0);
+  std::string attribute = "MuStructure"; // "ReadOutName"; // could come from .orcarc
```

Comment:
@jshlee - ReadOutName attribute is used to convert DD sensitive detectors to G4 ones. Please, check that this change does not affect GEM sim hits.